battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Placing.java

```diff
@@ -8,13 +8,13 @@
 /**
  * Created by egolesor on 18.07.16.
  */
-public class PlacingResponse {
+public class Placing {
```

Comment:
`Placing` is a verb, you know? So is `Shoot`. Are you certain you wanted your classes named like that?